### DISCUSSION TITLE
The future of Fief — Big changes and roadmap update 🏰

### DISCUSSION THREAD

**Discussion by frankie567** [fief-dev/fief#322] [Announcements]
This January, we celebrate 2 years of Fief 🎉 When I started this project, my goal was to build an authentication platform that would be **open**, **secure**, and **quick and easy** to integrate into any project.

At the time of writing, Fief has more than **350 stars on GitHub** and **240 active workspaces** on the public Cloud beta, serving **2000 users**. Over two years, more than 100 questions have been raised in our [GitHub Discussions forum](https://github.com/orgs/fief-dev/discussions). Not so bad for a solo-project in a domain counting already very good and mature project!

Thanks to the numerous feedback we had, I can confidently say that we have a **fresh and innovative vision** on what a user authentication system should be. However, I also see that I overestimated the importance of some features and aspects. The most important one being: **the cloud offer**.

# [Read more](https://polar.sh/fief-dev/posts/the-future-of-fief)

**Comment by ocontant** [comment]
I agree with you that Cloud offering was a hard sell. You are competing with very mature free offering that are used to upsell Cloud services infrastructure like AWS, GCP, Microsoft Azure. Their authentication offering is hard to compete with.

But if you want to have an AirGap or offline solution, you cannot use Cloud providers, as such you need something you can self host. This could be a Niche market for you, if you sell training, support and consulting services for Fief (The usual Open Source monetization path).

In that regards, It would be a nice thing to have SSO support, to authenticate against Active Directory, LDAP, and Kerberos, to support enterprise context.

**Reply by frankie567** [reply]
Indeed! I think our main challenge will be to make it really easy to setup Fief on any server or hosting you like.

And yes, having support for professional directories is also something that would be great!

**Comment by JacobCoffee** [comment]
Any plans (or are you open to PRs) for adding native support for other frameworks apart from FastAPI and Flask such as Sanic, Litestar, etc?

**Reply by frankie567** [reply]
Not in the short term, but open to proposals! Don't know much about Sanic or Litestar, but I suppose they have similar concepts to FastAPI and Flask; so porting the existing integrations should be quite straightforward :)
